src/core/utils/format-address.ts

```diff
@@ -1,19 +1,31 @@
 import { Blockchain } from '../blockchain/types';
 import { getBlockchain } from '../blockchain/blockchain-factory';
 
+const stripAddress = (text: string) => {
+    if (text.length === 0) {
+        return '';
+    }
+    let convertedStr = '';
+    convertedStr += text.substring(0, 5);
+    convertedStr += '.'.repeat(3);
```

Comment:
This is the definition of lazy :)) You don't type two more `.` you type `.repeat(3)` :))